scss/components/_menu.scss

```diff
@@ -34,40 +38,29 @@ $menu-icon-spacing: 0.25rem !default;
 /// @type Color
 $menu-item-background-hover: $light-gray !default;
 
-/// Color for bordered menu
-/// @type Color
-$menu-border: $light-gray !default;
-
 /// Creates the base styles for a Menu.
 @mixin menu-base {
-  margin: $menu-margin;
-  list-style-type: none;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+  position: relative;
 
   @if $global-flexbox {
     display: flex;
     flex-wrap: nowrap;
-    align-items: center;
-    width: 100%;
   }
 
-  // List items are table cell to allow for vertical alignment
-  > li {
+  li {
     @include disable-mouse-outline;
-
-    @if $global-flexbox {
-      flex: 0 0 auto;
-    }
-    @else {
-      display: table-cell;
-      vertical-align: middle;
-    }
   }
 
-  // Reset line height to make the height of the overall item easier to calculate
-  > li > a {
-    display: block;
-    padding: $menu-item-padding;
+  a,
+  .button {
```

Comment:
All links and buttons inside a menu are this way? That feels a little heavy-handed... though I suppose this is low enough specificity we can override if we need to... @brettsmason what do you think?